src/composables/UserAuthentication.js

```diff
@@ -73,6 +78,11 @@ export function completeSilentLogin() {
  * to fetch the user's information from the authentication provider.
  * @return {Promise<Object>} Promise with the current user on success otherwise an error.
  */
-export function getUser() {
-  return userManager.getUser();
+export async function getUser() {
+  return userManager.getUser().then((user) => new User({
```

Comment:
what if userManager is null ?
![image](https://github.com/ditrit/leto-modelizer/assets/62500081/010d14fc-6f73-4875-be77-3e630aed5429)